src/erlcloud_ddb_util.erl

```diff
@@ -51,7 +51,7 @@
 -type range_key_name() :: erlcloud_ddb2:range_key_name().
 -type table_name() :: erlcloud_ddb2:table_name().
 
--type items_return() :: {ok, [out_item()]}
+-type items_return() :: {ok, [out_item() | binary()]}
```

Comment:
Curious—where do these binary values come from in the client? I didn't realize DDB supported returning a list of strings.

Reply:
In this specific case, they come from tests `test/erlcloud_ddb_util_tests.erl:665` and `test/erlcloud_ddb_util_tests.erl:812`. In goes a list of `binary()` out comes another one. Do you think the test is wrong (if I change the type `dialyzer` starts complaining again).

Reply:
Those tests both look like lazy BS garbage-in/garbage out for test cases that test functions that treat those values opaquely in the layer tested. Could you just change this to

```erlang
    Item1 = [{<<"key">>, <<"item_1">>}],
    Item2 = [{<<"key">>, <<"item_2">>}],
```

in both cases? Thanks.

Reply:
> Could you just change this

Sure, here it is: a76e395. Tests are passing.

Also, since now (?) `dialyzer` is complaining about the non-opaqueness of `meck:expect`'s 4th argument, I decided to bump it to 0.9.0 (latest stable release), that solves that issue with https://github.com/eproxus/meck/issues/212. Here it is: 6933ee7.

This surfaced a new issue that I solved in afe0067.

Tests are passing.